signin.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="KO">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Linkbrary</title>
+    <link rel="stylesheet" href="/style.css">
+</head>
+<body>
+    <div class="main-position">
+        <nav>
+            <a href="/"><img src="./image/logo.png" alt="linkbrary logo" width="210.58px" height="38px"></a>
+        </nav>
+        <div class="join-us">
+            회원이 아니신가요? <a href="./signup.html">회원가입하기</a>
+        </div>
+        <div>
+            <form>
+                <div class="sign-title">
+                    이메일
+                </div>
+                <div>
+                    <input class="input-case" name="email" type="email">
+                </div>
```

Comment:
input을 설명해주고 있는 텍스트는 label 태그를 이용해서 작성해주셔야 하는데요~
label의 for, input의 id도 연결시켜주셔야 해요!

```suggestion
                <label class="sign-title" for="email">
                    이메일
                </label>
                <div>
                    <input class="input-case" id="email" name="email" type="email">
                </div>
```